Clinical-Genomics/cg
# Adds Turn-Around-Time column  to cli status cases

This PR adds estimated turnaround time (last sample received to last sample delivered) to cli/status/cases

How to test:
0. install on rasta stage 
1. run cg status cases -o date

Expected outcome:
* New TAT column
* cases without delivery-date and should have an parentesis indicating its an estimate
* cases with received date and delivered date have tat as the days between those dates
* cases lacking a date have a schablon value of 
** received to prepared 4 days
** prepared to sequenced 5 days
** sequenced to analysed 4 days
** analysed to uploaded 1 day
** uploaded to delivered 2 days 
* the schablon values are calculated in the script calculate_avg_tat.sql
* colorised case rows:
** cases with received date and delivered date and tat <= 21 are green
** cases with estimated tat == 21 are yellow
** cases with estimated/real tat > 21 are red 
** the rest of the cases are white
